app/build.gradle

```diff
@@ -113,7 +113,7 @@ dependencies {
       'com.chaos.view:pinview:1.4.4',
       'com.github.bumptech.glide:glide:4.11.0',
       'com.google.android:flexbox:2.0.1',
-      'com.google.android.material:material:1.2.0-alpha02',
+      'com.google.android.material:material:1.4.0-alpha02',
```

Comment:
Reason of updating - https://github.com/material-components/material-components-android/commit/fd848e8aaceee6178577fd3af9fe07609158d402